app/assets/javascripts/controllers/deploy_groups_ctrl.js

```diff
@@ -1,9 +1,13 @@
 samson.controller('DeployGroupsCtrl', function($scope, $http, $location, $window) {
 
   function init() {
-    $http.get($location.path() + '/deploys.json').success(function(result) {
+    $http.get($location.path() + '.json').success(function(result) {
       result.deploys.forEach(function(deploy) {
-        deploy.start = new Date(deploy.started_at);
+        if (deploy.started_at != undefined) {
+          deploy.start = new Date(deploy.started_at);
+        } else {
+          deploy.start = new Date(deploy.created_at);
+        }
```

Comment:
I think you could condense that to `deploy.start = new Date(deploy.started_at || deploy.created_at)`.